assets/js/main.js

```diff
@@ -221,6 +224,7 @@ function updateVoidTraderInventory() {
         const itemString = `<tr><td>${currentItem.item}</td>` +
                   `<td>${currentItem.ducats}</td><td>${currentItem.credits}</td></tr>`;
         $('#voidTraderInventoryContent').append(itemString);
+        $('a[href$="#voidTraderInventoryPanel"').click(() => pkry.packery());
```

Comment:
Shouldn't that be just `updateGrid()`?

Reply:
other sources said packery, but i'll try that instead.

Reply:
No, I mean just `updateGrid()`. That's function exposed from Packery closure.